### PR TITLE
Af 70 customer authorization

### PR DESCRIPTION
**Pull request by TrofDanil**

### CODE REVIEW

File: src/test/java/tests/TestCustomerAuthirization.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+import pages.CustomerPage;
+
+@Log4j
+public class TestCustomerAuthirization extends TestInit {
+
+    CustomerLoginPage customerLoginPage;
+    CustomerPage customerPage;
+    String USER_FIRST_NAME = "Jack";
+    String USER_LAST_NAME = "the Ripper";
+    String USER_NAME = USER_FIRST_NAME + USER_LAST_NAME;
```
Comment (StepaniukAndii) [inline src/test/java/tests/TestCustomerAuthirization.java]: data types should be private

Comment (StepaniukAndii) [reply src/test/java/tests/TestCustomerAuthirization.java]: and lowercase

**Review by StepaniukAndii — DISMISSED** [review]

File: src/test/java/tests/TestCustomerAuthirization.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import pages.CustomerPage;
+
+@Log4j
+public class TestCustomerAuthirization extends TestInit {
+
+    CustomerLoginPage customerLoginPage;
+    CustomerPage customerPage;
+    private final static String first_name = "Jack";
+    private final static String last_name = "the Ripper";
+    private final static String user_name = first_name + last_name;
```
Comment (StepaniukAndii) [inline src/test/java/tests/TestCustomerAuthirization.java]: camel case)

File: src/test/java/tests/TestCustomerAuthirization.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+import org.testng.annotations.Test;
+
+import pages.CustomerLoginPage;
+import pages.CustomerPage;
+
+@Log4j
+public class TestCustomerAuthirization extends TestInit {
+
+    CustomerLoginPage customerLoginPage;
+    CustomerPage customerPage;
```
Comment (StepaniukAndii) [inline src/test/java/tests/TestCustomerAuthirization.java]: private !!!!

**Review by AlexandrYaitskiyi18 — APPROVED** [review]

**Pull request merged**